### DISCUSSION TITLE
How to suppress auto update for Dax Studio

### DISCUSSION THREAD

**Discussion by Saijanjam** [DaxStudio/DaxStudio#1214] [General]
Kindly, please help me how to suppress auto update for Dax Studio.

**Comment by marcosqlbi** [comment]
Use the Block Version Checks setting in the Options.

![image](https://github.com/DaxStudio/DaxStudio/assets/13840816/157853ff-2dbf-4b97-af31-59037d1d931f)

**Reply by Saijanjam** [reply]
Hi marcosqlbi, I am unable to find options, can you help me please?
![image](https://github.com/DaxStudio/DaxStudio/assets/103157184/39b58ca5-35cb-4090-bc9f-b16f9c434799)

**Reply by Saijanjam** [reply]
Can I suppress Auto Updates for Dax through Registry? If yes, please let me knw. Or Even Through .json file?

**Comment by marcosqlbi** [comment]
File / Options / search "Update" in the search box.

**Reply by Saijanjam** [reply]
Can you please help me how to suppress those updates through powershell?

**Reply by Saijanjam** [reply]
Thanks Mate, I found it. 
reg: HKEY_CURRENT_USER\SOFTWARE\DaxStudio\ ' BlockVersionChecks' 'True'

**Reply by dgosbell** [reply]
Note that DAX Studio does not auto update. It just shows a notification when an update is available. 

If you are going to block these checks you should manually check for updates on a regular basis. Most of the recent bug reports we've had are for issues fixed in the latest release.

**Reply by Saijanjam** [reply]
Thank you
